stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -141,11 +146,9 @@ spec:
           path: /etc/machine-id
           type: File
     {{- end }}
-{{- if .Values.backend.es.tls_ca }}
-      - name: es-tls-secret
+      - name: fluentbit-secret
```

Comment:
also, this doesn't sound correct

Reply:
Any suggestions?

Reply:
I've separated secrets into two - one used by es (also see pr https://github.com/helm/charts/pull/14849/files which added more overrides for es secrets) and another for gcp.